corehq/apps/es/es_query.py

```diff
@@ -194,6 +194,11 @@ def add_filter(*args, **kwargs):
                 return add_filter
         raise AttributeError("There is no builtin filter named %s" % attr)
 
+    def __getitem__(self, sliced):
+        start = sliced.start or 0
+        size = sliced.stop - start
+        return self.start(start).size(size).run()
```

Comment:
(was looking at [python docs](https://docs.python.org/2/reference/datamodel.html#object.__getitem__) "For sequence types, the accepted keys should be integers and slice objects.")